src/types/transaction.ts

```diff
@@ -43,6 +43,10 @@ export interface SendTransactionParams {
   maxRoundsToWaitForConfirmation?: number
   /** **WARNING**: Not recommended for production use due to https://github.com/algorand/go-algorand/issues/5914. Whether to use simulate to automatically populate app call resources in the txn objects. Defaults to true when there are app calls in the group.  */
   populateAppCallResources?: boolean
+  /** The first round this transaction will be valid */
+  firstValid?: number
+  /** The last round this transaction will be valid */
+  lastValid?: number
```

Comment:
Yeah good point. Let me know what you think of [623edb0](https://github.com/algorandfoundation/algokit-utils-ts/pull/292/commits/623edb0e3a0396ae9b89c37b5331e07b88f72f7e)

Reply:
Looks good, noting most methods also allow you to pass in suggested params as an override to avoid the algod call, but maybe that detail doesn't matter